{
  "id": "CVE-2016-5598",
  "assigner": "secalert_us@oracle.com",
  "Published": "2016-10-25T12:31:09.75Z",
  "Modified": "2018-05-02T13:20:24.41Z",
  "last-modified": "2018-05-02T13:20:24.41Z",
  "summary": "Unspecified vulnerability in the MySQL Connector component 2.1.3 and earlier and 2.0.4 and earlier in Oracle MySQL allows remote attackers to affect confidentiality, integrity, and availability via vectors related to Connector/Python.",
  "access": {
    "authentication": "NONE",
    "complexity": "MEDIUM",
    "vector": "NETWORK"
  },
  "impact": {
    "availability": "PARTIAL",
    "confidentiality": "PARTIAL",
    "integrity": "PARTIAL"
  },
  "cvss": 6.8,
  "exploitabilityScore": 8.6,
  "impactScore": 6.4,
  "cvss-vector": "AV:N/AC:M/Au:N/C:P/I:P/A:P",
  "references": [
    "http://www.oracle.com/technetwork/security-advisory/cpuoct2016-2881722.html",
    "http://www.securityfocus.com/bid/93653",
    "http://www.securitytracker.com/id/1037050"
  ],
  "vulnerable_configuration": [
    "cpe:2.3:a:oracle:mysql_connector\\/python:2.0.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:oracle:mysql_connector\\/python:2.0.4:*:*:*:*:*:*:*",
    "cpe:2.3:a:oracle:mysql_connector\\/python:2.1.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:oracle:mysql_connector\\/python:2.1.3:*:*:*:*:*:*:*"
  ],
  "vulnerable_product": [
    "cpe:2.3:a:oracle:mysql_connector\\/python:2.0.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:oracle:mysql_connector\\/python:2.0.4:*:*:*:*:*:*:*",
    "cpe:2.3:a:oracle:mysql_connector\\/python:2.1.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:oracle:mysql_connector\\/python:2.1.3:*:*:*:*:*:*:*"
  ],
  "vendors": [
    "oracle"
  ],
  "products": [
    "mysql_connector\\/python"
  ],
  "vulnerable_product_stems": [
    "cpe:2.3:a:oracle:mysql_connector\\/python"
  ],
  "vulnerable_configuration_stems": [
    "cpe:2.3:a:oracle:mysql_connector\\/python"
  ],
  "cwe": "CWE-284",
  "vulnerable_configuration_cpe_2_2": []
}